index.js

```diff
@@ -32,6 +33,15 @@ Object.assign(ylem, {
 	propertyDecorators,
 });
 
+const observe = makeObserve.observe;
+
+makeObserve.observe = function(input) {
```

Comment:
Personally, I think we should just make it possible to turn off 
function return values from being proxied.  Ylem should just extend `observe.Object` and `observe.Array` in a way that this functionality is turned off and then export these things.

I'm not sure `.map()` results such should __ever__ try to turn its results into observables.  `DefineList` does this, but I'm not convinced it's right.  I'm very ok turning this off (for now optionally, later by default) in `can-observe`.

Reply:
I agree, I like the idea of exporting `ObserveObject` and `ObserveArray` with the functionality turned off. While the fix in this PR works, it seems kinda hacky just being in the `index.js` file.